src/components/MultiSelectCombobox/MultiSelectCombobox.js

```diff
@@ -0,0 +1,256 @@
+import React, { useState, useRef, useEffect } from "react";
+import { Label } from "@trussworks/react-uswds";
+import PillButton from "../PillButton/PillButton";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCaretDown, faCheck } from "@fortawesome/free-solid-svg-icons";
+import "./MultiSelectCombobox.scss";
+
+const getComboboxEnabledItems = (arr) => {
+  return arr.filter((e) => e.enabled);
+};
+
+const MultiSelectCombobox = ({
+  items,
+  label,
+  entity,
+  onChangeUpdate,
+  searchBy,
+}) => {
+  const [filter, setFilter] = useState("");
+  const [_items, _setItems] = useState(getComboboxEnabledItems(items));
+  const [data, setData] = useState(
+    JSON.parse(JSON.stringify(getComboboxEnabledItems(items)))
+  );
+  const [showListBox, setShowListBox] = useState(false);
+  const [selectedItems, setSelectedItems] = useState([]);
+  const selectedItemsRef = useRef(selectedItems);
+  const [stillMounted, setStillMounted] = useState(true);
+
+  useEffect(() => {
+    populateSelectedItems();
+    return () => setStillMounted(false);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
populateSelectedItems is going to be recreated on every render. Even if it doesn't change, a new one exists on every instance of the component.

There's no reason for this eslint ignore, it should contain [populateSelectedItems] and the effect can be moved below all the declared functions.

Reply:
Updated